site/cypress/integration/ui_tests/e2e_tests/01-site-navigation.spec.js

```diff
@@ -61,6 +61,16 @@ describe("Navigating the site and reading articles", function() {
         })
     });
 
+    it('An error in an article without roles', function () {
+      cy.visit(globalTestData.deepArticlePath);
+      cy.get(article.roles).should('have.text', 'Error please notify Field Staff Support there are no roles associated with this article');
+    });
+
+    it('A list of roles in an article', function () {
```

Comment:
Name is misleading. This is not testing a list. There should be tests for a single item (as here), a moderate sized list and a huge list.

Reply:
done